provider/openstack/local_test.go

```diff
@@ -105,19 +107,18 @@ func registerLocalTests() {
 type localServer struct {
 	Openstack       *openstackservice.Openstack
 	Nova            *novaservice.Nova
+	Neutron         *neutronservice.Neutron
```

Comment:
Nova and Neutron are necessary.  Openstack is needed for control of the http services for the test double.